### PR TITLE
rename telemetry.auto.version to telemetry.distro.version and add telemetry.distro.name

### PR DESCRIPTION
**Pull request by zeitlinger**
as defined in https://opentelemetry.io/docs/specs/semconv/resource/#telemetry-sdk-experimental

### CODE REVIEW

**Review by mateuszrzeszutek — APPROVED** [review]
👍

**Review by trask — APPROVED** [review]

File: smoke-tests/src/test/groovy/io/opentelemetry/smoketest/AppServerTest.groovy [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 package io.opentelemetry.smoketest
 
+
```
Comment (laurit) [inline smoke-tests/src/test/groovy/io/opentelemetry/smoketest/AppServerTest.groovy:8]:
```suggestion

```

**Comment by trask** [comment]
@zeitlinger can you update the PR title? thx

**Comment by zeitlinger** [comment]
> @zeitlinger can you update the PR title? thx

done

**Pull request merged**